src/StatisticBlock/schema.js

```diff
@@ -1,98 +1,206 @@
 import animationSchema from './animationSchema';
 import stylesSchema from './stylesSchema';
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  StatisticItem: {
+    id: 'Statistic item',
+    defaultMessage: 'Statistic item',
+  },
+  Default: {
+    id: 'Default',
+    defaultMessage: 'Default',
+  },
+  Value: {
+    id: 'Value',
+    defaultMessage: 'Value',
+  },
+  Label: {
+    id: 'Label',
+    defaultMessage: 'Label',
+  },
+  ExtraInfo: {
+    id: 'Extra info',
+    defaultMessage: 'Extra info',
+  },
+  Link: {
+    id: 'Link',
+    defaultMessage: 'Link',
+  },
+  StatisticBlock: {
+    id: 'Statistic block',
+    defaultMessage: 'Statistic block',
+  },
+  Animation: {
+    id: 'Animation',
+    defaultMessage: 'Animation',
+  },
+  Styling: {
+    id: 'Styling',
+    defaultMessage: 'Styling',
+  },
+  Horizontal: {
+    id: 'Horizontal',
+    defaultMessage: 'Horizontal',
+  },
+  CanPresentItsMeasurementHorizontally: {
+    id: 'Can present its measurement horizontally',
+    defaultMessage: 'Can present its measurement horizontally',
+  },
+  Inverted: {
+    id: 'Inverted',
+    defaultMessage: 'Inverted',
+  },
+  CanBeFormattedDarkBackground: {
+    id: 'Can be formatted to fit on a dark background.',
+    defaultMessage: 'Can be formatted to fit on a dark background.',
+  },
+  Size: {
+    id: 'Size',
+    defaultMessage: 'Size',
+  },
+  Mini: {
+    id: 'Mini',
+    defaultMessage: 'Mini',
+  },
+  Tiny: {
+    id: 'Tiny',
+    defaultMessage: 'Tiny',
+  },
+  Small: {
+    id: 'Small',
+    defaultMessage: 'Small',
+  },
+  Large: {
+    id: 'Large',
+    defaultMessage: 'Large',
+  },
+  Huge: {
+    id: 'Huge',
+    defaultMessage: 'Huge',
+  },
+  Columns: {
+    id: 'Columns',
+    defaultMessage: 'Columns',
+  },
+  One: {
+    id: 'One',
+    defaultMessage: 'One',
+  },
+  Two: {
+    id: 'Two',
+    defaultMessage: 'Two',
+  },
+  Three: {
+    id: 'Three',
+    defaultMessage: 'Three',
+  },
+  Four: {
+    id: 'Four',
+    defaultMessage: 'Four',
+  },
+  StatisticItems: {
+    id: 'Statistic items',
+    defaultMessage: 'Statistic items',
+  },
+  Styles: {
+    id: 'Styles',
+    defaultMessage: 'Styles',
+  },
+});
 
 const statisticSchema = {
-  title: 'Statistic item',
+  title: intl.formatMessage(messages.StatisticItem),
```

Comment:
@macagua intl is not defined

Reply:
@avoinea I am sorry for the delay, please can you check again this "pull request" and if all ok merge it? thanks

Reply:
@avoinea Done with commit [269b52f](https://github.com/eea/volto-statistic-block/pull/9/commits/269b52f084104604f0087ec84bcc8c928021f487)